fv3gfs-util/fv3gfs/util/partitioner.py

```diff
@@ -68,9 +68,9 @@ def global_extent(self, rank_metadata: QuantityMetadata) -> Tuple[int, ...]:
     @abc.abstractmethod
     def subtile_slice(
         self,
-        rank: int,
         global_dims: Sequence[str],
         global_extent: Sequence[int],
+        rank: int,
```

Comment:
Please keep the same ordering of the arguments, in case the ML team is depending on this ordering.

Reply:
Reverted the argument order change.